src/main/java/com/ffxivcensus/gatherer/Gatherer.java

```diff
@@ -36,27 +35,15 @@
     public void run() {
         try {
             LOG.debug("Starting evaluation of player ID: " + getPlayerId());
-            // Parse players and write them to DB
-            PlayerBean player = PlayerBuilder.getPlayer(getPlayerId(), 1);
-            // Currently ignore deleted characters (404)
-            if(!CharacterStatus.DELETED.equals(player.getCharacterStatus())) {
-                String out = getDao().saveRecord(player);
-                LOG.debug(out);
-            }
-            RESULT_LOG.info(getPlayerId() + " - " + player.getCharacterStatus().name());
-        } catch(MySQLNonTransientConnectionException failWriteEx) { // If record fails to write due to too many connections
-            LOG.trace("Error: Record write failure, reattempting write");
-            // Wait a second
-            try {
-                Thread.currentThread().wait(1);
-            } catch(InterruptedException e) {
-            }
-            // Then attempt to write again
-            try {
-                String out = getDao().saveRecord(PlayerBuilder.getPlayer(getPlayerId(), 1));
-                LOG.debug(out);
-            } catch(Exception e) {
-                LOG.error(e.getMessage(), e);
+
+            // Check whether we already know about this character
+            PlayerBean player = playerRepository.findOne(getPlayerId());
+            if(player == null || !CharacterStatus.DELETED.equals(player.getCharacterStatus())) {
```

Comment:
When the repository returns `null`, we've never seen this character before, so we definitely want to check them. Otherwise, pick up non-Deleted characters.